try:
    print(1/0)
except Exception:
    print("未知的错误")

